src/utils/createComponent.ts

```diff
@@ -98,4 +98,18 @@ export function createComponent<I extends HTMLElement, E extends EventNames = {}
         }
       : options,
   );
+
+  return React.forwardRef<typeof component, Parameters<typeof component>[0]>((props, ref) => {
```

Comment:
I wonder if we could somehow avoid using `forwardRef`? It adds `Anonymous` wrapper to each component in React DevTools tree (and we already have two classes per component instance there):
<img width="305" alt="Screenshot 2023-12-13 at 15 33 30" src="https://github.com/vaadin/react-components/assets/10589913/8df2f731-e5ee-4865-bdda-899de9264ee2">

Passing `hidden: Boolean` to the `elementClass` in case of Polymer workaround fixes it for me. 
I'm not sure if that's a good idea though as there might be other boolean attributes.

Reply:
> Passing hidden: Boolean to the elementClass in case of Polymer workaround fixes it for me.

Thanks, let's go with this solution for now because of how simple it is and since we haven't identified problematic properties other than `hidden`.